packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -68,6 +66,8 @@ export default class ContentSwitcher extends React.Component {
     size: PropTypes.oneOf(['sm', 'md', 'lg', 'xl']),
   };
 
+  static contextType = PrefixContext;
```

Comment:
Nothing wrong here, just curious why some components need to use context? Is it bc they're class components?

Reply:
Yeah, exactly! Class components can't use hooks so they have to grab context via the more old school this.context method